rust-src/curve_arithmetic/src/bls12_381_instance.rs

```diff
@@ -360,9 +359,7 @@ impl Curve for G2Affine {
 
     fn generate_scalar<T: Rng>(csprng: &mut T) -> Self::Scalar { Fr::random(csprng) }
 
-    fn hash_to_group(_b: &[u8]) -> Self {
-        unimplemented!("hash_to_group_element for G2Affine of Bls12_381 is not implemented")
-    }
+    fn hash_to_group(b: &[u8]) -> Self { hash_to_curve_g2(b, HASH_TO_GROUP_G1_DST).into_affine() }
```

Comment:
```suggestion
    fn hash_to_group(b: &[u8]) -> Self { hash_to_curve_g2(b, HASH_TO_GROUP_G2_DST).into_affine() }
```

I think this looks like a copy-paste error.